store/tikv/2pc.go

```diff
@@ -477,6 +477,7 @@ func (c *twoPhaseCommitter) buildPrewriteRequest(batch batchKeys) *tikvrpc.Reque
 			LockTtl:           c.lockTTL,
 			IsPessimisticLock: isPessimisticLock,
 			ForUpdateTs:       c.forUpdateTS,
+			TxnSize:           uint64(len(batch.keys)),
```

Comment:
I think use  `len(c.keys)` is better. It is possible that many batches write to a same region because we limit each batch not exceed 16KB.

Reply:
You also should consider this case that a batch write whose index keys are discrete, maybe one region per index key.

Reply:
How about set the `bigTxnThreshold` a smaller number like 16?

Reply:
ok